Signal/src/ViewControllers/BlockListViewController.m

```diff
@@ -137,18 +137,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 - (NSString *)displayNameForIndexPath:(NSIndexPath *)indexPath
 {
     NSString *phoneNumber = _blockedPhoneNumbers[(NSUInteger)indexPath.item];
-    PhoneNumber *parsedPhoneNumber = [PhoneNumber tryParsePhoneNumberFromUserSpecifiedText:phoneNumber];
-
-    // Try to parse and present the phone number in E164.
-    // It should already be in E164, so this should always work.
-    // If an invalid or unparsable phone number is already in the block list,
-    // present it as-is.
-    NSString *displayName = (parsedPhoneNumber ? parsedPhoneNumber.toE164 : phoneNumber);
-    Contact *contact = self.contactMap[displayName];
-    if (contact && [contact fullName].length > 0) {
-        displayName = [contact fullName];
-    }
-
+    NSString *displayName = [_contactsManager displayNameForPhoneIdentifier:phoneNumber];
```

Comment:
Do you see that in the previous implementation of displayNameForIndexPath there was a situation where we would return phoneNumber?

It seems like there no longer is.